src/main/cli/cli.c

```diff
@@ -306,18 +308,6 @@ static const char *configurationStates[] = {
     [CONFIGURATION_STATE_CONFIGURED] = "CONFIGURED"
 };
 
-typedef enum dumpFlags_e {
-    DUMP_MASTER = (1 << 0),
-    DUMP_PROFILE = (1 << 1),
-    DUMP_RATES = (1 << 2),
-    DUMP_ALL = (1 << 3),
-    DO_DIFF = (1 << 4),
-    SHOW_DEFAULTS = (1 << 5),
-    HIDE_UNUSED = (1 << 6),
-    HARDWARE_ONLY = (1 << 7),
-    BARE = (1 << 8),
-} dumpFlags_t;
-
 static void cliExit(const bool reboot);
 typedef bool printFn(dumpFlags_t dumpMask, bool equalsDefault, const char *format, ...);
```

Comment:
Then how could it be used inside unit tests?

It is used in every `print*` function in `cli.c`.

Reply:
The `printFn` should be moved together with dumpFlags_t. Ideally, it would be in `cli_unittest.h` or `cli_impl.h` so that it does not pollute cli.h file (neither dumpFlags_t nor printFn is intended to be public interface)

Reply:
I mean how `printAux` and others should be tested?

Reply:
Sorry, I overlooked this usage. It may be reasonable to publish.

Personally, i'd move to `cli_impl.h` - header with declarations used internally by CLI that are not public interface, but are necessary to communicate between multiple files implementing cli (there is single cli.c now, but unittest is IMO eligible).